Schema/milestone.schema.json

```diff
@@ -35,73 +41,53 @@
                 "open",
                 "closed"
             ],
-            "default": "open"
+            "default": "open",
+            "required": true
         },
         "title": {
             "type": "string",
-            "description": "The title of the milestone."
+            "description": "The title of the milestone.",
+            "required": true
         },
         "description": {
-            "type": [
-                "string",
-                "null"
-            ]
+            "type": "string",
+            "nullable": true,
+            "required": true
         },
         "creator": {
-            "anyOf": [
-                {
-                    "type": "null"
-                },
-                {
-                    "$ref": "https://vng.opencatalogi.nl/schemas/simpleUser.schema.json"
-                }
-            ]
+            "$ref": "https://vng.opencatalogi.nl/schemas/simpleUser.schema.json",
+            "nullable": true,
+            "required": true
         },
         "open_issues": {
-            "type": "integer"
+            "type": "integer",
+            "required": true
         },
         "closed_issues": {
-            "type": "integer"
+            "type": "integer",
+            "required": true
         },
         "created_at": {
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "required": true
         },
         "updated_at": {
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "required": true
         },
         "closed_at": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "date-time"
+            "type": "string",
+            "format": "date-time",
+            "nullable": true,
+            "required": true
         },
         "due_on": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "date-time"
+            "type": "string",
+            "format": "date-time",
+            "nullable": true,
+            "required": true
         }
-    },
-    "required": [
-        "closed_issues",
-        "creator",
-        "description",
-        "due_on",
-        "closed_at",
-        "id",
-        "node_id",
-        "labels_url",
-        "html_url",
-        "number",
-        "open_issues",
-        "state",
-        "title",
-        "url",
-        "created_at",
-        "updated_at"
-    ]
+    }
```

Comment:
I think choosing the option of a required array, instead of every property was the better way and just leaving out the ones that are nullable. 

Is there a reason for doing so? Please let me know.